monai/inferers/merger.py

```diff
@@ -274,23 +274,23 @@ def __init__(
             shape=self.merged_shape,
             chunks=self.chunks,
             dtype=self.output_dtype,
-            compressor=self.compressor,
+            compressor=None,
```

Comment:
Hi @kolasaniv1996, thanks for the fix!
Coudl you please add a `deprecated_arg` on this `compressor`, since 1.5 and removed in 1.7?
Also could you please also add `codecs` which suggested from zarr format 3?
https://zarr.readthedocs.io/en/stable/api/zarr/api/asynchronous/index.html#zarr.api.asynchronous.create